### PR TITLE
Update DEM

### PR DESCRIPTION
**Pull request by cmarshak**
Utilize the updated dem-stitcher API and fixing issues related to #65.

Will include sample products over:

- [x] La palma
- [x] Aleutian
- [x] California
- [x] pair from #60

### CODE REVIEW

**Comment by cmarshak** [comment]
The resulting comparisons were done using existing products on the ASF

California:

V5: S1-GUNW-D-R-115-tops-20220504_20220422-141610-00124W_00038N-PP-0442-v2_0_5
V6: [S1-GUNW-D-R-115-tops-20220504_20220422-141610-00124W_00038N-PP-0442-v2_0_6](https://dem-updates-to-gunw-v206.s3.us-west-2.amazonaws.com/S1-GUNW-D-R-115-tops-20220504_20220422-141610-00124W_00038N-PP-0442-v2_0_6.nc)
command: `isce2_topsapp --reference-scenes S1A_IW_SLC__1SDV_20220504T141557_20220504T141624_043062_05246D_3C67 --secondary-scenes S1A_IW_SLC__1SDV_20220422T141557_20220422T141624_042887_051EA4_CD2E > topsapp_img.out 2> topsapp_img.err`

Aleutian chain:

v5: S1-GUNW-D-R-044-tops-20210925_20200930-172127-00169W_00053N-PP-812d-v2_0_5
V6: [S1-GUNW-D-R-044-tops-20210925_20200930-172127-00169W_00053N-PP-812d-v2_0_6](https://dem-updates-to-gunw-v206.s3.us-west-2.amazonaws.com/S1-GUNW-D-R-044-tops-20210925_20200930-172127-00169W_00053N-PP-812d-v2_0_6.nc)
Command: `isce2_topsapp --reference-scenes S1A_IW_SLC__1SDV_20210925T172118_20210925T172137_039841_04B68A_7D82 --secondary-scenes S1A_IW_SLC__1SDV_20200930T172045_20200930T172114_034591_0406F4_8C2F S1A_IW_SLC__1SDV_20200930T172112_20200930T172131_034591_0406F4_17D7 > topsapp_img.out 2> topsapp_img.err`

La Palma:

V5: S1-GUNW-D-R-169-tops-20220520_20220508-071114-00020W_00027N-PP-f4ed-v2_0_5.nc
V6: [S1-GUNW-D-R-169-tops-20220520_20220508-071114-00020W_00027N-PP-f4ed-v2_0_6.nc]((https://dem-updates-to-gunw-v206.s3.us-west-2.amazonaws.com/S1-GUNW-D-R-169-tops-20220520_20220508-071114-00020W_00027N-PP-f4ed-v2_0_6.nc)
Command: `isce2_topsapp --reference-scenes S1A_IW_SLC__1SDV_20220520T071101_20220520T071128_043291_052B7B_7B63 --secondary-scenes S1A_IW_SLC__1SDV_20220508T071100_20220508T071127_043116_05262A_AA24 > topsapp_img.out 2> topsapp_img.err`

Issue 60 over Tibet:

V5: S1-GUNW-A-R-158-tops-20220308_20220224-123254-00080E_00035N-PP-4300-v2_0_5
V6: S1-GUNW-A-R-158-tops-20220308_20220224-123254-00080E_00035N-PP-4300-v2_0_6
command: `isce2_topsapp --reference-scenes S1A_IW_SLC__1SDV_20220320T123241_20220320T123308_042405_050E5B_B237 --secondary-scenes S1A_IW_SLC__1SDV_20220308T123241_20220308T123307_042230_05086E_4B83`

**Comment by cmarshak** [comment]
All the above links and data have been updated using the latest version of Dockerized TopsApp.

**Review by sssangha — APPROVED** [review]

**Comment by cmarshak** [comment]
After Sim/Joe's review on the related issue in `dem-stitcher` (in the [issue](https://github.com/ACCESS-Cloud-Based-InSAR/DockerizedTopsApp/issues/65) above there is a referenced related/coupled `dem-stitcher` [issue](https://github.com/ACCESS-Cloud-Based-InSAR/dem-stitcher/pull/33)), we released the newest version of PyPI that resolves the `dem-stitcher` piece and since Sim's review on _this_ repository, I have updated the environment file to point to this new release of `dem-stitcher`. I removed all version ranges as it appears the `rasterio` release on `conda-forge` is working as expected without a requirement to specify an older `gdal` version. I also did cosmetic changes to the `environment.yml` for this repository as recommended by Joe in `dem-stitcher` such as no capitalization of packages.

After re-installing the environment (i.e. removing it and then re-installing all the packages as indicated by the `environment.yml`), I re-ran the workflow on the Aleutian example above and it ran to completion. I am re-uploaded the product to s3 and the relevant product link above is update. For clarity, [here](https://gunw-development-testing.s3.us-west-2.amazonaws.com/dem-updates-May-2022/S1-GUNW-D-R-044-tops-20210925_20200930-172127-00169W_00053N-PP-812d-v2_0_6.nc) it is too.

In short, the few commits since the last review should _not change the outputs inspected above_.
